Feat: get started with AI Gateway (#577)
Kong/developer.konghq.com: app/_how-tos/get-started-with-ai-gateway.md

---
title: Get started with AI Gateway
content_type: how_to

products:
    - gateway

works_on:
    - on-prem
    - konnect

plugins:
  - ai-proxy

entities: 
  - service
  - route
  - plugin

tags:
    - get-started
    - ai-gateway

tldr: 
  q: What is AI Gateway, and how can I get started with it?
  a: |

>     With Kong's AI Gateway, you can deploy AI infrastructure for traffic 
>     being sent to one or more LLMs, which lets you semantically route, secure, observe, accelerate, 
>     and govern using a special set of AI plugins that are bundled with {{site.base_gateway}} distributions.

Reviewer: ```suggestion
    With Kong's AI Gateway, you can deploy AI infrastructure for traffic 
    that is sent to one or more LLMs. This lets you semantically route, secure, observe, accelerate, 
    and govern traffic using a special set of AI plugins that are bundled with {{site.base_gateway}} distributions.
```
